src/main/groovy/jenkins/automation/utils/CommonUtils.groovy

```diff
@@ -19,4 +24,31 @@ class CommonUtils {
             }
         }
     }
+
+/**
+ * Utility function to add extended email
+ *
+ * @param triggers List<String> triggers E.g Failure, Fixed etc...
+ * @param String emails Comma separated string of emails
+ * @param sendToDevelopers Default false,
+ * @param includeCulprits Default false,
+ * @param sendToRecipientList Default true
+ *
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#common-utils" target="_blank">BDD job Example</a>
+ */
+
+    static void extendedEmail(context, List<String> triggers = ["Failure", "Fixed"], String emails, sendToDevelopers = false, includeCulprits = false, sendToRecipientList = true) {
```

Comment:
ah, well then. Here are my 2 cents:

I think we should change the API to support a string, and only a string.

Yes, there will be some work for us to change any existing configs, but it's a few minutes of work and probably not a big deal.

I'll merge this version in now, and then we can plan a time -- probably next week -- with all interested parties to make the cutover from the List to a string.

I think we should be trying to write a) as little code as possible, b) code that doesn't deviate from job-dsl for no good reason